dev.rdm.RDM.metainfo.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Consider making it part of your code repository as it's not specific to Flathub.